pkg/plugin/canary/pause.go

```diff
@@ -107,14 +108,14 @@ func NewCmdUnpause(streams genericclioptions.IOStreams) *cobra.Command {
 	return cmd
 }
 
-// Complete sets all information required for processing the command
-func (o *PauseOptions) Complete(cmd *cobra.Command, args []string) error {
```

Comment:
we used `Complete()` `Validate()` `Run()` to be compatible with an Interface definition to ease testing and mocking

```golang
type Cmd interface {
  Complete(cmd *cobra.Command, args []string) error
  Validate() error 
  Run() error
}

Reply:
I see, thanks for the context. Let's keep them private for now as they don't need to be public. It will be easy to switch back to public when we want to implement the interface